src/translations/en.json

```diff
@@ -498,8 +498,18 @@
         "filtered_by_device": "device: {device_name}",
         "filtered_by_area": "area: {area_name}"
       },
+      "file-upload": {
+        "uploading": "Uploading...",
+        "uploading_name": "Uploading {name}",
+        "label": "Drop your file here, or {browse}",
+        "browse": "browse",
+        "unsupported_format": "Unsupported format, please choose a JPEG, PNG, or GIF image."
+      },
       "picture-upload": {
-        "label": "Picture",
+        "label": "Drop your picture here, or {browse}",
```

Comment:
I think the bigger label should be kept succinct, and move directions to the secondary text:
- Label: "Add a picture"
- Secondary: "Drag and drop a file here or click to browse for one. Supported formats are JPEG, PNG, or GIF."

Reply:
@matthiasdebaat

Reply:
I've suggested an alternative here https://github.com/home-assistant/frontend/pull/17654#discussion_r1307223948